docs/dev/stream/state/checkpointing.md

```diff
@@ -76,6 +76,8 @@ Other parameters for checkpointing include:
 
   - *fail/continue task on checkpoint errors*: This determines if a task will be failed if an error occurs in the execution of the task's checkpoint procedure. This is the default behaviour. Alternatively, when this is disabled, the task will simply decline the checkpoint to the checkpoint coordinator and continue running.
 
+  - *is prefer checkpoint for recovery*: This determines if a job will fallback to checkpoint when there is a savepoint later than checkpoint.
```

Comment:
I would change the wording slightly here to be more grammatically correct:

*prefer checkpoint for recovery*: This determines if a job will fallback to latest checkpoint even when there are more recent savepoints available to potentially reduce recovery time